docs/advanced_topics/settings.rst

```diff
@@ -292,6 +292,14 @@ Email Notifications format
 
 Notification emails are sent in `text/plain` by default, change this to use HTML formatting.
 
+Email Notifications to superusers
+---------------------------------
+
+.. code-block:: python
+
+  WAGTAILADMIN_NOTIFICATION_INCLUDE_SUPERUSERS = False
+
+Notification emails are sent to moderators and superusers by default. You can change this to exclude superusers and only notify moderators.
```

Comment:
Perhaps we could mention that superusers who do really want to get emails can also be added to the "Moderators" group?

Reply:
Hmm... I'm concerned that documenting that as a feature, rather than it simply being a current implementation quirk, might be taking us down a bit of a murky road...

Up to now, Wagtail has never properly addressed the distinction between "publish" as a permission and "moderation" as a role. Technically, Wagtail has no such concept as "a moderator"; there's a group called "Moderators" in the out-of-the-box setup, whose sole distinguishing feature is that they have 'publish' permission, but this is just a convention, and it's entirely valid to delete, rename or edit that group. Essentially, describing a user as a moderator is a convenient fiction, that (hopefully) helps people to understand the workflow system, but in reality it just means "someone who has publish permission over the given page". This equivalence breaks down when we have users such as developers, who have publish permission but shouldn't really be assigned the _role_ of moderator.

This PR is a first pass at addressing that issue, relying on the assumption that the users in that category will be superusers. This isn't perfect - as mentioned at https://github.com/wagtail/wagtail/pull/3972#issuecomment-339424381, I want to leave open the possibility of having "almost-superuser" groups with an administrative role - but it's better than what we have now.

Since "the Moderators group" is not a fixed thing, making the statement "superusers who are in the Moderators group will receive notifications" really unpacks as:

* Users who have received publish permission through _being members of a group_ will receive notifications;
* Users who have received publish permission through _being superusers_ will not receive notifications.

The fact that publish permission has different effects depending on how you acquired it is a bug, not a feature. I think the only real way to solve it _properly_ is to make the role of "moderation" a real thing that's distinct from "having publish permission". That's not going to be solved in the short term, and until we do, I think we should avoid making promises about its behaviour at that level of detail.

Reply:
Hi guys. I appreciate your concerns regarding making it a feature that people might rely upon and that it might make it harder to drop later.

As a superuser of a small scale Web app, where we wanted to use Wagtail to manage our static pages, I was surprised to see that content approval was including me by default. Maybe it was mostly a documentation issue? I write the code, not the content, and English isn't my first language.

From reading this thread, I'm under the impression that @gasman is leaning towards removing this 'notify super-users' and rely only upon permissions & group for content review, am I getting this right? 

I'm more than happy to change the implementation of this pull request and I would also understand if you decline this change and leave the code as is.

Whatever you decide, please ping me if you would like any action from me on this.

Reply:
This is something were are also running into as the group that should be empowered with approving moderation is actually small but we have admins and others who have publish permissions, but should not be approving requests from users who need moderation.

What are thoughts on perhaps having a setting that exposes what groups should receive moderation messages, instead of just excluding superusers?  It seems like it would work to solidify the moderator role.